docs/3.0rc/deploy/infrastructure-concepts/deploy-via-python.mdx

```diff
@@ -0,0 +1,385 @@
+            work_pool_name="my-work-pool",
+            image="my-docker-image:dev",
+            push=False,
+            # Run once a minute
+            interval=timedelta(minutes=1)
+        )
+    ```
+    </Expandable>
+
+- `cron`
+
+    Defines when a flow should run using a cron string.
+    <Expandable title="Example">
+    ```python cron.py
+    from prefect import flow
+
+    @flow(log_prints=True)
+    def my_flow(name: str = "world"):
+        print(f"Hello, {name}!")
+
+    if __name__ == "__main__":
+        my_flow.deploy(
+            name="my-deployment",
+            work_pool_name="my-work-pool",
+            image="my-docker-image:dev",
+            push=False,
+            # Run once a day at midnight
+            cron="0 0 * * *"
+        )
+    ```
+    </Expandable>
+
+- `rrule`
+
+    Defines a complex schedule using an `rrule` string.
+
+    <Expandable title="Example">
+    ```python rrule.py
+    from prefect import flow
+
+    @flow(log_prints=True)
+    def my_flow(name: str = "world"):
+        print(f"Hello, {name}!")
+
+    if __name__ == "__main__":
+        my_flow.deploy(
+            name="my-deployment",
+            work_pool_name="my-work-pool",
+            image="my-docker-image:dev",
+            push=False,
+            # Run every weekday at 9am
+            rrule="FREQ=WEEKLY;BYDAY=MO,TU,WE,TH,FR;BYHOUR=9;BYMINUTE=0"
+        )
+    ```
+    </Expandable>
+
+- `schedules`
+
+    Defines multiple schedules for a deployment. This option provides flexibility for:
+        - Setting up various recurring schedules
+        - Implementing complex scheduling logic
+        - Applying timezone offsets to schedules
+
+    <Expandable title="Example">
+    ```python schedules.py
+    from datetime import datetime, timedelta
+    from prefect import flow
+    from prefect.client.schemas.schedules import IntervalSchedule
+
+    @flow(log_prints=True)
+    def my_flow(name: str = "world"):
+        print(f"Hello, {name}!")
+
+    if __name__ == "__main__":
+        my_flow.deploy(
+            name="my-deployment",
+            work_pool_name="my-work-pool",
+            image="my-docker-image:dev",
+            push=False,
+            # Run every 10 minutes starting from January 1, 2023
+            # at 00:00 Central Time
+            schedules=[
+                IntervalSchedule(
+                    interval=timedelta(minutes=10),
+                    anchor_date=datetime(2023, 1, 1, 0, 0),
+                    timezone="America/Chicago"
+                )
+            ]
+        )
+    ```
+    </Expandable>
+
+    Learn more about schedules [here](/3.0rc/automate/add-schedules).
+
+
+## Use remote code storage
+
+In addition to using Docker images, Prefect also supports deploying your code using remote storage. This approach allows you to 
+store your flow code in a remote location, such as a Git repository or cloud storage service.
+
+Using remote storage for your code has several advantages:
```

Comment:
loosely held nit:

wondering if we could add some statement to the effect of "github is where your code lives anyways" or something, bc often I notice people feel arbitrarily pigeonholed and forget github is an option

Reply:
Yeah, I can add something about why a remote git repository works well for code storage.